components/CheckDocumentation/connections/BlockingQuery.tsx

```diff
@@ -0,0 +1,145 @@
+        When any blocked query is running for longer than the specified
+        threshold of <code>{config.settings["warning_blocked_age_secs"]}</code>{" "}
+        seconds, creates an issue with severity "warning". Escalates to
+        "critical" if any blocked query continues to run longer than{" "}
+        <code>{config.settings["critical_blocked_age_secs"]}</code> seconds.
+        Resolves automatically once these queries complete and release their
+        locks.
+      </p>
+      <p>
+        Ignores any blocking queries that contain the{" "}
+        <code>/* pganalyze:no-alert */</code> or{" "}
+        <code>/* pganalyze=no-alert */</code> magic comment.
+      </p>
+    </>
+  );
+};
+
+const BlockingQueryGuidance: React.FunctionComponent<CheckGuidanceProps> = ({
+  urls: { backendsUrl, SettingLink },
+  issue,
+}) => {
+  const Link = useSmartAnchor();
+  const issueDetails = issue && JSON.parse(issue.detailsJson);
+  const showAutovacuum =
+    !issueDetails || issueDetails["has_autovacuum_to_prevent_wraparound"];
+  return (
+    <div>
+      <h4>Impact</h4>
+      <p>
+        Queries that hold contended locks for a significant period of time may
+        end up preventing other queries from making progress. This can even
+        block simple SELECT queries, and in some cases, it can affect overall
+        system performance or it can cause the system to run out of connections.
+      </p>
+      <h4>Common Causes</h4>
+      <ul>
+        {showAutovacuum && (
+          <li>
+            <h5>Autovacuum (to prevent wraparound)</h5>
+            <p>
+              In general, autovacuum won't block other queries. If a query tries
+              to acquire a lock that conflicts with a running autovacuum,
+              autovacuum will be interrupted so that the query can proceed.
+              However, if autovacuum is running to prevent transaction ID
+              wraparound, it will not yield like this and will block other
+              queries trying to acquire conflicting locks.
+            </p>
+          </li>
+        )}
+        <li>
+          <h5>Long-running migration</h5>
+          <p>
+            If some of these queries are DDL that is part of a database
+            migration, they may need to rewrite a table (e.g., changing the type
+            of a column) or scan a whole table (e.g., <code>CREATE INDEX</code>{" "}
+            without <code>CONCURRENTLY</code>), which can take much longer in
+            production than in a staging or development environment. If your
+            migration is transactional, consider canceling it and refactor it to
+            avoid table rewrites.
+          </p>
+        </li>
+        <li>
+          <h5>Other long-running queries</h5>
+          <p>
+            A long running query could cause many blocking queries when that
+            query is holding locks that conflict with many other queries.
```

Comment:
This first sentence is a little wordy... Maybe just

```
            A long running query could block other queries if it holds locks that they require to make progress.
```

or something like that?